index.js

```diff
@@ -57,6 +80,49 @@ function assembleStyles() {
 		});
 	});
 
+	function rgb2rgb(r, g, b) {
+		return [r, g, b];
+	}
+
+	styles.color.close = '\u001b[39m';
+	styles.bgColor.close = '\u001b[49m';
+
+	styles.color.ansi = {};
+	styles.color.ansi256 = {};
+	styles.color.ansi16m = {
+		rgb: wrapAnsi16m(rgb2rgb, 0)
+	};
+
+	styles.bgColor.ansi = {};
+	styles.bgColor.ansi256 = {};
+	styles.bgColor.ansi16m = {
+		rgb: wrapAnsi16m(rgb2rgb, 10)
+	};
+
+	for (var key in colorConvert) {
+		if (!colorConvert.hasOwnProperty(key)) {
+			continue;
+		}
+
+		var fn = colorConvert[key];
+		var conversion = key.replace(/^([^2]+).+$/, '$1');
```

Comment:
> [from][to]

I don't get it. Can't you convert in both directions? So shouldn't it have both?

I would be fine with two loops if it got rid of the regex stuff.